website/versioned_docs/version-0.73/intro-react.md

```diff
@@ -132,7 +132,7 @@ export default Cat;
 
 You can think of curly braces as creating a portal into JS functionality in your JSX!
 
-> Because JSX is included in the React library, it won’t work if you don’t have `import React from 'react'` at the top of your file!
+> If you have React Native version 0.64 AND React Version 17 or later, you no longer **need**to `import React from 'react'`,
```

Comment:
took a look and it turns out I'd missed the news :sweat_smile: TypeScript wasn't complaining, but [ESLint was](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md). 

a couple of suggestions for the notice:
- since this is the 0.73 article, you can remove the 0.64 mention
- minor grammar/formatting tweaks

```suggestion
> If you're using React Native with React 17 or later, you **no longer need** to `import React from 'react'`. Find out more [here](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)
```

you'll also want to apply this to the 0.74 (`website/versioned_docs/version-0.74/intro-react.md`) and main (`docs/intro-react.md`) versions of this article so your changes don't get lost in the future.

Reply:
That looks wonderful. Thank you.